src/Data/Attribute.php

```diff
@@ -30,7 +30,7 @@ public function __construct(string $key, array $values = [])
 
     public function addValue(mixed $value): void
     {
-        DataHelper::checkAttributeValueNotExceedingCharacterLimit($this->getKey(), $value);
+        DataHelper::checkAttributeValueNotExceedingCharacterLimit($this->getKey(), (string) $value);
```

Comment:
:wrench: This change should be accompanied by a unit test.

Reply:
Ok, thought the integration test would be enough